plugins/inputs/internetSpeedMonitor/internetSpeedMonitor_test.go

```diff
@@ -0,0 +1,55 @@
+package internetSpeedMonitor
+
+import (
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInternetConfiguration(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping network-dependent test in short mode.")
+	}
+	speedMonitor := &SpeedMonitor{
+		EnableFileDownload: false,
+		Measurement:        "internet_speed",
+	}
+
+	assert.Equal(t, speedMonitor.Measurement, "internet_speed")
+	assert.Equal(t, speedMonitor.EnableFileDownload, false)
+	assert.Equal(t, speedMonitor.Description(), "Monitors internet speed in the network")
+}
```

Comment:
Well this basically tests golangs ability to set structure members. :-) I don't think we should test it here.